ProjFS.Mac/PrjFSKext/PrjFSService.cpp

```diff
@@ -1,7 +1,7 @@
 #include "public/PrjFSCommon.h"
 #include "public/PrjFSProviderClientShared.h"
 #include "PrjFSService.hpp"
-#include "PrjFSProviderUserClient.hpp"
+#include "PrjFSProviderUserClientPrivate.hpp"
```

Comment:
No need to change this now (and I don't like adding yet another naming scheme) but it looks a bit weird to include the "Private.cpp" of `PrjFSProviderUserClient` in `PrjFSService.cpp`.

Elsewhere the "Private.hpp" naming scheme has meant that the contents of the .hpp should only be known to the .cpp with the same name.

Perhaps something like "PrjFSProviderUserClientKernel.hpp"?

cc: @jeschu1